release-notes.txt

```diff
@@ -1,3 +1,751 @@
+  lookups (#17510)
++ sys/stdio_nimble: add new stdio module using nimble (#12012)
+* drivers/atwinc15x0: enable sleep state (#17854)
+* drivers/encx24j600: define default parameters (#17747)
+* gcoap: multi-transport support (#16688)
+* gnrc/ipv6nib: remove the need for evtimer-minutes (#17411)
+* gnrc/netif: convert to ztimer (#17354)
+* ipv6/nib: bugfix of 6CO length checking (#17850)
+* ipv6/nib: fix memcpy() bug in _handle_rtr_timeout() (#17741)
+* lwip: enable LWIP_SO_RCVTIMEO if sock layer is used (#17779)
+* pkg/lwip: use ztimer_msec instead of xtimer (#17115)
+* pkg/semtech-loramac: enable setting channels mask (#17824)
+* pm: don't (un)block IDLE mode [backport 2022.04] (#17975)
+* sys/gnrc/sixlowpan/frag/fb: guard sfr_types.h header include (#17730)
+* sys/net/gnrc/sock: use ztimer_usec or xtimer for timeout (#17852)
+* sys/net/gnrc/sock_types: guard tcp.h header inclusion (#17743)
+* sys/net/gnrc: ztimer_no_periph_rtt if gomach (#17729)
+* sys/net/gnrc_sixlowpan_frag_sfr: use xtimer_set (#17668)
+* sys/net/link_layer/csma_sender: use public xtimer_now() (#17731)
+* sys/net/sock_util: fix compilation for IPv4-only mode (#17766)
+
+Packages
+--------
++ buildsystem: add EXTERNAL_PKG_DIRS functionality (#17211)
++ pkg/etl: Add the embedded template library (etl) (#17477)
++ pkg/lvgl: add extra widget dependency (#17760)
++ pkg/nanors: add reed solomon codec implementation (#17703)
++ pkg/tflite-micro: add support and deprecate tensorflow-lite (#17908)
++ pkg/tinydtls: add IPv4 support (#17765)
++ pkg/uwb-dw1000: add patch for rf_txctrl value (#17724)
++ pkg/wamr: add WAMR to provide WASM support in RIOT (#15329)
+* buildsystem/pkg: expand packages from USEPKG early as path into
+  PKG_PATHS (#17551)
+* make: pkg.mk: don't call `git am` if there are no patches (#17858)
+* pkg/edhoc-c: bump version (#17777)
+* pkg/littlefs2: bump version to 2.4.2 (#17837)
+* pkg/lvgl: bump to 8.2.0 (#17681)
+* pkg/lvlgl: allow cusomizing LV_MEM_SIZE (#17759)
+* pkg/mynewt-core: fix semaphore (#17771)
+* pkg/tinycbor: bump version (#17604)
+* pkg/tinydtls/sock_dtls: use ztimer_usec (#17677)
+* pkg/tinydtls: migrate to ztimer64_msec (#17564)
+
+Boards
+------
+
++ boards/adafruit-pybadge: add support (#17807)
++ boards/b-l475e-iot01a: Add SPI2 and SPI3 buses (#17885)
++ boards/nucleo-f429zi: add support for ethernet (#17828)
++ boards: add support for nucleo-f439zi (#17827)
++ cpu/esp32: update to Espressif's precompiled ESP32 vendor toolchain
+  with gcc 8.4.0 (#17769)
++ cpu/stm32: add backup battery monitoring (VBAT) (#16989)
++ drivers: add driver for L3GD20H 3-axis gyroscope (#10082)
++ sys/auto_init: add auto_init_leds, drop LED init code from boards (#17584)
++ sys/board_common: add generic board_init() function (#17008)
++ sys/vfs: add file-system auto-mount (#17341)
++ sys/vfs: add vfs_default, configure default fs for same54-xpro (#17643)
++ sys/ztimer: add auto_adjust module (#17633)
```

Comment:
All these should go to sys

Reply:
I think the
```
sys/auto_init: add auto_init_leds, drop LED init code from boards (#17584)
sys/board_common: add generic board_init() function (#17008)
sys/vfs: add vfs_default, configure default fs for same54-xpro (#17643)
```
also make sense in the boards section.